integrationTests/longTests/storage_test.go

```diff
@@ -0,0 +1,102 @@
+package longTests
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWriteContinuously(t *testing.T) {
+	if testing.Short() {
+		t.Skip("this is not a short test")
+	}
+
+	nbTxsWrite := 1000000
+	testStorage := integrationTests.NewTestStorage()
+	store := testStorage.CreateStorageLevelDB()
+
+	defer func() {
+		_ = store.DestroyUnit()
+	}()
+
+	startTime := time.Now()
+	for i := 1; i <= nbTxsWrite; i++ {
+		written := 10000
```

Comment:
You might want to move `written := 10000` before the `for` loop.

Reply:
+1

Reply:
Done.